core/src/main/java/fj/data/NonEmptyList.java

```diff
@@ -35,12 +33,26 @@
   /**
    * The first element of this linked list.
    */
-  public A head() { return head; }
+  public A head() {
+    return head;
+  }
 
   /**
    * This list without the first element.
    */
-  public List<A> tail() { return tail; }
+  public List<A> tail() {
+    return tail;
+  }
+
+  public List<A> init()
```

Comment:
Trivial: Prefer the `{` on the same line as the method name.  Applies elsewhere.